Assignment6/app.py

```
@@ -0,0 +1,207 @@
+        save_to_cache(ticker, company_data, stock_data)
+        
+        # Save to search history
+        save_search_history(ticker)
+        
+        return {
+            'company': company_data,
+            'stock': stock_data,
+            'from_cache': False
+        }
+        
+    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)